tests/test_overrides_parser.py

```diff
@@ -1953,3 +1954,309 @@ def test_sweep_iterators(
     ]
     assert actual_sweep_string_list == expected_sweep_string_list
     assert actual_sweep_encoded_list == expected_sweep_encoded_list
+
+
+# Dummy assignments to allow copy-paste of `TEST_CONFIG_DATA` below from
+# OmegaConf, and properly identify parsing error cases.
+# (should be replaced by imports from OmegaConf in the future)
+GrammarSyntaxError = HydraException
+# The OmegaConf exceptions below cannot be identified at parsing time.
+ConfigKeyError = GrammarTypeError = UnsupportedInterpolationType = None
+
+# DO NOT EDIT `TEST_CONFIG_DATA`!
+# It should be copied regularly from OmegaConf to ensure Hydra's grammar is able
+# to parse similar expressions as OmegaConf's.
+TEST_CONFIG_DATA: List[Tuple[str, Any, Any]] = [
```

Comment:
this test file is already massive.
Can you put the OmegaConf tests in a different test file?

Reply:
Good idea, done in 044a006f2